src/Configuration.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  Configuration.swift
+//  Sango
+//
+//  Created by Adam Shea on 5/7/18.
+//  Copyright © 2018 Afero, Inc. All rights reserved.
+//
+
+import Foundation
+
+public struct Configuration {
+    
+    var inputAssets:String
```

Comment:
standard swift is to have a space between the colon and type. "var inputAssets: String"